tailnet/proto/convert.go

```diff
@@ -0,0 +1,139 @@
+package proto
```

Comment:
there is already a `tailnet/convert.go` --- add these there or move the existing conversions here. We should just have one place; I don't mind which.